app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError(
+                f"Wow, {visitor['name']} is so late (cafe.py, line 16)."

```

Comment:
`(cafe.py, line 16)` For what purpose?:))

Reply:
Well, a good example in the checklist has 'raise CustomError("We have an error on the line 16.")'
I had to change the number 3 times after updating the file for flake8 recommendations. It wasn't fun :sweat_smile: